src/preferences/pages/general.ts

```diff
@@ -1,121 +1,87 @@
-        GTypeName: 'RoundedWindowCornersPrefsGeneral',
-
-        // Widgets export from template ui
+        GTypeName: 'PrefsGeneral',
         InternalChildren: [
-            'global_settings_preferences_group',
-            'enable_log_switch',
-            'skip_libadwaita_app_switch',
-            'skip_libhandy_app_switch',
-            'tweak_kitty_switch',
-            'preferences_entry_switch',
-            'border_width_ajustment',
-            'border_color_button',
-            'edit_shadow_row',
-            'applications_group',
-            'reset_preferences_btn',
+            'skip_libadwaita',
+            'skip_libhandy',
+            'border_width',
+            'border_color',
+            'corner_radius',
+            'corner_smoothing',
+            'keep_for_maximized',
+            'keep_for_fullscreen',
+            'paddings',
+            'tweak_kitty',
+            'right_click_menu',
+            'enable_log',
         ],
     },
-    class extends Gtk.Box {
-        private declare _global_settings_preferences_group: Gtk.ListBox;
-        private declare _enable_log_switch: Gtk.Switch;
-        private declare _skip_libhandy_app_switch: Gtk.Switch;
-        private declare _skip_libadwaita_app_switch: Gtk.Switch;
-        private declare _tweak_kitty_switch: Gtk.Switch;
-        private declare _preferences_entry_switch: Gtk.Switch;
-        private declare _border_width_ajustment: Gtk.Adjustment;
-        private declare _border_color_button: Gtk.ColorButton;
-        private declare _edit_shadow_row: Gtk.ListBoxRow;
-        private declare _applications_group: Gtk.ListBox;
-        private declare _reset_preferences_btn: Gtk.Button;
-
-        private config_items!: _Items;
-
-        _init() {
-            super._init();
-
-            this.config_items = new RoundedCornersItem();
-
-            this.build_ui();
-
-            connections
-                .get()
-                .connect(
-                    settings().g_settings,
-                    'changed',
-                    (_settings: Gio.Settings, key: string) =>
-                        this._on_settings_changed(key),
-                );
+    class extends Adw.PreferencesPage {
+        private declare _skip_libadwaita: Adw.SwitchRow;
+        private declare _skip_libhandy: Adw.SwitchRow;
+        private declare _border_width: Gtk.Adjustment;
+        private declare _border_color: Gtk.ColorDialogButton;
+        private declare _corner_radius: Gtk.Adjustment;
+        private declare _corner_smoothing: Gtk.Adjustment;
+        private declare _keep_for_maximized: Adw.SwitchRow;
+        private declare _keep_for_fullscreen: Adw.SwitchRow;
+        private declare _paddings: PaddingsRowClass;
+        private declare _tweak_kitty: Adw.SwitchRow;
+        private declare _right_click_menu: Adw.SwitchRow;
+        private declare _enable_log: Adw.SwitchRow;
+
+        private declare _cfg: RoundedCornersCfg;
+
+        constructor() {
```

Comment:
This entire constructor feels like a wall of text, I thing you should add newlines between different `set_value`->`connect` blocks

Reply:
Fixed in 932515bb184b43c528d264bf4fa1872fc73c622f